Tools/cases_generator/generate_cases.py

```diff
@@ -176,12 +214,28 @@ def main():
             file=sys.stderr,
         )
     with eopen(args.output, "w") as f:
-        write_cases(f, instrs, supers)
+        effects_table = write_cases(f, instrs, supers, families)
     if not args.quiet:
         print(
             f"Wrote {ninstrs + nsupers} instructions to {args.output}",
             file=sys.stderr,
         )
+    # Check that families have consistent effects
```

Comment:
Families don't have consistent stack effects, because some of the members are effectively superinstructions, like `BINARY_OP_INPLACE_ADD_UNICODE`.

I'd drop this for now, until we can define those family members as custom superinstructions.

Reply:
Alternatively, mark special cases like `BINARY_OP_INPLACE_ADD_UNICODE` and skip them here?